spec/features/events_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.feature "Event Applicant Overview", :type => :feature do
+  scenario "logged in as Pupil can not see overview" do
+    @event = FactoryGirl.create(:event)
+    @profile = FactoryGirl.create(:profile)
+    @profile.user.role = :pupil
+    login_as(@profile.user, :scope => :user)
+    visit event_path(@event)
```

Comment:
The above code is nearly identical in all this tests. Maybe it can be factored out?